docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+      corporate:
+        price: $500
+      independent:
+        price: $275
+      student:
+        price: $100
+
+* ``sponsorship``: the conference sponsorship prices. The keys are fixed and
+  referred to in other places. Example::
+
+    sponsorship:
+      first_draft:
+        price: $900
+      second_draft:
+        price: $2,500
+      publisher:
+        price: $5,000
+      patron:
+        price: $9,500
+      keystone:
+        price: $17,000
+
+* ``sponsors``: confirmed conference sponsors, using the same keys as the
+  pricing. Each sponsor has a name and a link to their website. Their logo
+  should be stored in ``_static/img/sponsors/``. Example::
+
+    sponsors:
+      keystone:
+      patron:
+        # logo should be in _static/img/sponsors/patron-sponsor.jpg/png
+        - name: patron-sponsor
+          link: http://www.example.com
+      publisher:
+      second:
+      first:
+      media:
+        - name: media-sponsor-one
+          link: http://www.example.com
+
+* ``date``: the conference dates and days. Contains:
+    * ``main``: the human readable conference dates and location,
+      e.g. ``"**May 3-5, 2020, in Portland, Oregon**"``.
+    * ``short``: the short human readable dates, e.g. ``May 3-5, 2020``.
+    * ``tickets_live``: a rough human readable date when tickets go on sale,
+      e.g. ``January 2020``.
+    * ``month``: the month in which the conference is held, e.g. ``May``.
+    * ``total_talk_days``: the number of days that have talks, e.g. ``2``.
+      Used to automatically read the schedule.
+    * ``day_one``, ``day_two``, etc. These are actually events, not days. Each "day" has:
+        * ``event``: the name of the event, like ``Hike``, ``Writing Day`` or
+          ``Main Conference``.
+        * ``date``: the short human readable date, e.g. ``May 2`` or
+          ``May 4-5``.
+        * ``summary``: a human readable summary of the event that day.
+        * ``icon``: the icon used for this event, e.g. ``hike`` or
+          ``conference``.
+        * ``dotw``: the day(s) of the week for this event, e.g. ``Saturday``
+          or ``Monday/Tuesday``.
+* ``about``: general conference background. Contains:
+    * ``attendees``: the number of attendees.
+    * ``summary``: a summary text for the conference.
+    * ``venue``: a human readable textual description of the venue location.
+    * ``photos``: a link to the conference photos (typically on Flickr).
+    * ``mainroom``: the name of the main room.
+    * ``unconfroom``: the name of the unconference room.
+    * ``projector_ratio``: the ratio for the projector, e.g. ``16:9``.
+* ``cfp``: call for papers details. Contains:
+    * ``url``: the URL to a google form with the CFP.
+    * ``ends``: a human readable date of when the CFP ends,
+      e.g. ``31 January, 2020``.
+    * ``notification``: a human readable date of when accepted speakers
+      will be notified.
+* ``job_fair.location``: a description of the location of the job fair.
```

Comment:
Example? Is this just a room name? Maybe move this to the section that shows the other rooms....